### DISCUSSION TITLE
Wordpress search / TSML connection

### DISCUSSION THREAD

**Discussion by k8fish3r** [code4recovery/12-step-meeting-list#895] [General]
Hi,
I wondered how site owners using TSML - handle the gap between "Wordpress search" not able to see TSML - data?

Looking at searches it definitely looks like our site members don't understand the disconnect.

Wonder if i could borrow some thoughts.

IF

Kate F

**Comment by tim-rohrer** [comment]
Hi Kate,

So the Wordpress search on my dev site works fine for finding meetings. What is your site URL? Assuming you've tested yourself, what results are you seeing?

If it isn't working for meetings, have you check to see if the Wordpress search finds other material on your site? The problem could be with your theme or Wordpress. You might have better luck in the help area for one of those.

You can also look in your browser's dev console to see if any errors are thrown.

Yis, Tim

**Reply by brianw-area24** [reply]
I check on my dev site, and the WordPress search appeared to be working there as well.

Brian W

**Reply by k8fish3r** [reply]
Hi,
revisiting whats happening. A more accurate ask would be - is there a way to omit certain parts of a meetings list from being interrogated when I site search occurs?

At present searches prioritize meeting information - as highly as content pages.
E.g. searching for content on traditions 
Will return every meeting which mentions traditions
Within the list will be the content page about traditions.

Is there a way to omit certain sections of meeting description? To reduce the clashing?
E.g. "meeting-notes" section.~

My thought if that isnt possible - might there be a way to stop site search returning meetings altogether. Not ideal but maybe better than content searches being flooded by meeting listings which happen to mention the same word within their description.

Thinking about it - I probably need another plugin which specialises in limiting what is searched.

Btw the website is codauk.org

**Reply by tim-rohrer** [reply]
Hi,

Yeah, we don’t do anything special with Wordpress search so I concur with your need to explore other options. 

Good luck!
